cinema/models.py

```diff
@@ -108,0 +1,111 @@
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.db import models
+
+
+class CinemaHall(models.Model):
+    name = models.CharField(max_length=255)
+    rows = models.IntegerField()
+    seats_in_row = models.IntegerField()
+
+    @property
+    def capacity(self) -> int:
+        return self.rows * self.seats_in_row
+
+    def __str__(self):
+        return self.name
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+
+    def __str__(self):
+        return self.name
+
+
+class Actor(models.Model):
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+
+    @property
+    def full_name(self):
+        return str(self)

```

Comment:
The `full_name` property should return the concatenated `first_name` and `last_name`, not `str(self)`. Consider using `return f'{self.first_name} {self.last_name}'`.